### DISCUSSION TITLE
Traefik.yml cloudflare certs not working

### DISCUSSION THREAD

**Discussion by contactnat** [techno-tim/techno-tim.github.io#238] [Questions]
a few of us over on Discord had issues with certs via cloudflare not working, adding the following line to the traefik.yml file worked: disablePropagationCheck: true

 By setting this flag to true disables the need to wait for the propagation of the TXT record to all authoritative name servers.
maybe you can update your template?

e.g.
certificatesResolvers:
  cloudflare:
    acme:
      email: myemail@gmail.com
      storage: acme.json
      dnsChallenge:
        provider: cloudflare
        disablePropagationCheck: true
        resolvers:
          - "1.1.1.1:53"
          - "8.8.8.8:53"

**Comment by timothystewart6** [comment]
Hey, nice catch! Thank you!

**Comment by lifeandfate1** [comment]
I found that "disablePropagationCheck: true" alone did not fix this issue and that adding a delay with "delayBeforeCheck: 60" enabled the certs to be generated without issue.

[Stuck on "Waiting for DNS record propagation"
](https://www.reddit.com/r/Traefik/comments/wysdxu/stuck_on_waiting_for_dns_record_propagation/)

That is a post on the Traefik subreddit listing this as the solution.

**Comment by contactnat** [comment]
> I found that "disablePropagationCheck: true" alone did not fix this issue and that adding a delay with "delayBeforeCheck: 60" enabled the certs to be generated without issue.
> 
> [Stuck on "Waiting for DNS record propagation" ](https://www.reddit.com/r/Traefik/comments/wysdxu/stuck_on_waiting_for_dns_record_propagation/)
> 
> That is a post on the Traefik subreddit listing this as the solution.

nice one, I had the same problem again several weeks ago and the fix that did last time didnt work this time, your addition did though, man this has been buggin me for ages and it now works, thanks for posting the fix!

**Comment by timothystewart6** [comment]
Should add this to the docs / repo!
